python/taichi/lang/_ndarray.py

```diff
@@ -97,6 +97,10 @@ def __getitem__(self, key):
         """
         raise NotImplementedError()
 
+    @python_scope
+    def release(self):
```

Comment:
IIUC we don't really need a explicit release, ndarray should follow python convention to destruct itself when it's EOL. (which calls into the cpp destructor)